apis/python/src/tiledbsoma/pytiledbsoma.cc

```diff
@@ -62,8 +62,9 @@ PYBIND11_MODULE(pytiledbsoma, m) {
     });
 
     py::enum_<OpenMode>(m, "OpenMode")
-        .value("read", OpenMode::read)
-        .value("write", OpenMode::write);
+        .value("soma_read", OpenMode::soma_read)
```

Comment:
Nit:   I do not object to the name change, but am wondering about the motivation.  Is the goal to distinguish the "soma_read" mode from some other "read" mode?  Seems redundant to me (at first glance).

Reply:
I did this because both `del` and `delete` are protected keywords and this is similar to how TileDB core writes its enum options (just lower case instead of all caps).

Reply:
ah, that makes sense!

If we are serious about this kind of error, strongly recommend enabling the flake8-builtins (ruff mode `A`) support in our ruff lint settings.